### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by ivywei0125**
What does this implement/fix? Explain your changes.
---------------------------------------------------
…

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-eventhubs/src/main/java/com/microsoft/azure/toolkit/intellij/eventhubs/IntellijEventHubsActionsContributor.java [hunk 1]
```diff
@@ -89,6 +90,16 @@ private void registerGroupCreateNamespaceActionHandler(AzureActionManager am) {
         am.registerHandler(EventHubsActionsContributor.GROUP_CREATE_EVENT_HUBS, (r, e) -> true, (r, e) -> handler.accept(r, (AnActionEvent) e));
     }
 
+    private void registerCopyNamespaceConnectionStringActionHandler(AzureActionManager am) {
+        final BiPredicate<EventHubsNamespace, AnActionEvent> condition = (r, e) -> true;
+        final BiConsumer<EventHubsNamespace, AnActionEvent> handler = (c, e) -> {
+            final String connectionString = c.getOrCreateListenConnectionString();
+            am.getAction(ResourceCommonActionsContributor.COPY_STRING).handle(connectionString);
+            AzureMessager.getMessager().info(AzureBundle.message("azure.eventhubs.info.copyConnectionString"));
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-eventhubs/src/main/java/com/microsoft/azure/toolkit/intellij/eventhubs/IntellijEventHubsActionsContributor.java:98]: add config action to open portal

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/right/MonitorLogTablePanel.form [hunk 2]
```diff
@@ -173,7 +173,7 @@
                 </constraints>
                 <properties>
                   <text value="Save as query"/>
-                  <toolTipText value="Save  Save the current settings filters "/>
+                  <toolTipText value="Save the current settings filters "/>
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/right/MonitorLogTablePanel.form:176]: remove tips

Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/right/MonitorLogTablePanel.form:176]: remove the tip or fix it.

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/action/IntellijAzureActionManager.java [hunk 3]
```diff
@@ -180,6 +180,14 @@ public ActionGroupWrapper(@Nonnull ActionGroup group) {
             this.addActions(group.getActions());
         }
 
+        @Override
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/action/IntellijAzureActionManager.java:183]: move getText to update method

Comment (ivywei0125) [reply PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/action/IntellijAzureActionManager.java:183]: add getActionUpdateThread

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/action/IntellijAzureActionManager.java [hunk 4]
```diff
@@ -180,6 +180,14 @@ public ActionGroupWrapper(@Nonnull ActionGroup group) {
...
+        public void update(@Nonnull AnActionEvent e) {
+            super.update(e);
+            final IView.Label view = this.group.getView();
+            final Presentation presentation = e.getPresentation();
+            Optional.ofNullable(view).ifPresent(v -> presentation.setIcon(IntelliJAzureIcons.getIcon(v.getIconPath())));
+        }
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/action/IntellijAzureActionManager.java:189]: move constructor logic here.

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/left/MonitorTreePanel.java [hunk 5]
```diff
@@ -89,8 +89,12 @@ private void addQueryNode(QueryData data) {
         if (isTableTab) {
             return;
         }
-        getOrCreateCustomQueriesTabNode().add(new DefaultMutableTreeNode(data));
-        this.customQueries.add(data);
+        final DefaultMutableTreeNode customQueryRootNode = getOrCreateCustomQueriesTabNode();
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/left/MonitorTreePanel.java:92]: should update tree model

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/right/filter/SaveFiltersAsQueryDialog.java [hunk 6]
```diff
@@ -45,11 +54,32 @@ public SaveFiltersAsQueryDialog(@Nullable Project project, String queryContent)
...
+        if (isExistingQuery(queryName) && !isOverride(queryName)) {
+            return;
+        }
         this.queryDataToSave.setDisplayName(queryNameField.getText());
         this.queryDataToSave.setQueryString(queryContent.getText());
         super.doOKAction();
     }
 
+    private boolean isExistingQuery(String queryName) {
+        final List<String> customQueryList = Optional.ofNullable(PropertiesComponent.getInstance()
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/right/filter/SaveFiltersAsQueryDialog.java:67]: read from tree panel list. do not read from propertiesComponent

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/action/ResourceCommonActionsContributor.java [hunk 7]
```diff
@@ -284,6 +285,12 @@ public void registerActions(AzureActionManager am) {
                 .withIcon(AzureIcons.Action.CREATE.getIconPath())
                 .visibleWhen(s -> s instanceof ServiceLinkerModule)
                 .withHandler((r) -> {
+                    if (r.getParent() instanceof SpringCloudDeployment) {
+                        final String appUrl = ((SpringCloudDeployment) r.getParent()).getParent().getParent().getPortalUrl();
+                        final String message = String.format("Please create Service Connector from {0} -- {1} -- {2} in <a href=\"%s\">Azure portal</a>.", appUrl);
```
Comment (ivywei0125) [inline Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/action/ResourceCommonActionsContributor.java:290]: add app name and modify -- to -->

**Review by wangmingliang-ms — APPROVED** [review]
LGTM, remember to resolve comments in later PRs.

**Pull request merged**